hack/verify/lint.sh

```diff
@@ -23,6 +23,8 @@ cd "${REPO_ROOT}"
 # build golangci-lint
 SOURCE_DIR="${REPO_ROOT}/hack/tools" hack/go_container.sh \
   go build -o /out/golangci-lint github.com/golangci/golangci-lint/cmd/golangci-lint
+SOURCE_DIR="${REPO_ROOT}/hack/tools" GOOS="linux" hack/go_container.sh \
```

Comment:
we specifically need a linux binary regardless of the host to do kindnetd

Reply:
Do we need to add judgment here?
Although it doesn't matter, it just takes a little more space for `golangci-lint-linux` and repeats it when the OS is Linux.